plugins-compat-tester/src/main/java/org/jenkins/tools/test/hook/PipelineStageViewHook.java

```diff
@@ -37,6 +37,6 @@ public static boolean isPipelineStageViewPlugin(Map<String, Object> moreInfo) {
     }
 
     public static boolean isPipelineStageViewPlugin(PomData data) {
-        return data.artifactId.contains("pipeline-rest-api");
+        return data.groupId.equals("org.jenkins-ci.plugins.pipeline-stage-view") || data.artifactId.contains("pipeline-rest-api");
```

Comment:
nit: I'd either add another case for `data.artifactId.contains("pipeline-stage-view")` or drop the case for `data.artifactId.contains("pipeline-rest-api")` for clarity. I think the groupId check by itself is good enough.

Reply:
The groupId works for version 2.13 but not for previous one, so the idea is to leave current behaviour until 2.12 and this versions and future one use the new logic in the hook